### ISSUE TITLE
False error on some comments after class closing brace

### ISSUE THREAD

**Issue by phlopsi** [squizlabs/PHP_CodeSniffer#689]
```
$ phpcs --version
PHP_CodeSniffer version 2.3.3 (stable) by Squiz (http://www.squiz.net)
```

My file "C1.php":
```php
<?php

namespace N1;

class C1
{
    
} // C1

```
This file should be correctly formatted according to PSR-2, but when i check on it i get this:
```
$ phpcs --standard=PSR2 C1.php

FILE: C:\Users\[...]\Desktop\C1.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 8 | ERROR | Closing class brace must be on a line by itself
----------------------------------------------------------------------

Time: 78ms; Memory: 2.25Mb
```

In PSR-2 it only says "*The opening brace for the class MUST go on its own line; the closing brace for the class MUST go on the next line after the body.*"
In the conclusion it's also mentioned that "*There are many elements of style and practice __intentionally omitted__ by this guide. These include but are not limited to: [...] Comments and documentation blocks*"

**Comment by gsherwood** [comment]
This doesn't have anything to do with the comment after the brace. The sniff already has specific code to allow this. It appears this is simple due to the space before the comment and the sniff not correctly ignoring whitespace as well as comment tokens.

**Comment by gsherwood** [comment]
Yep, that's all it was. Fix now committed. Thanks for reporting.
